modules/gapi/src/compiler/gstreaming.cpp

```diff
@@ -47,9 +52,7 @@ const cv::GMetaArgs& cv::GStreamingCompiled::Priv::outMetas() const
 // the G*Compiled's priv?
 void cv::GStreamingCompiled::Priv::setSource(cv::GRunArgs &&args)
 {
-    // FIXME: This metadata checking should be removed at all
```

Comment:
Is this fixme not applicable anymore?

Reply:
Yes it is - I decided it is ok to have this check here if the metadata has been passed